keps/sig-auth/4193-bound-service-account-token-improvements/README.md

```diff
@@ -360,9 +367,13 @@ you need any help or guidance.
 
 * `ServiceAccountTokenJTI` feature flag will toggle including JTI information in tokens, as well as recording JTIs in the audit log / the SA user info.
 * `ServiceAccountTokenPodNodeInfo` feature flag will toggle including node info associated with pods in tokens.
+* `ServiceAccountTokenNodeBindingValidation` feature flag will toggle the apiserver validating Node claims in node bound service account tokens.
 * `ServiceAccountTokenNodeBinding` feature flag will toggle allowing service account tokens to be bound to Node objects.
 
-Both of these feature flags can be disabled without any unexpected adverse affects or coordination required.
+The `ServiceAccountTokenNodeBindingValidation` feature will graduate to beta one release earlier than `ServiceAccountTokenNodeBinding`
+to ensure a safe rollback from version N+1 to N (more info below in rollback considerations section).
+
+All other feature flags can be disabled without any unexpected adverse affects or coordination required.
 
 ###### How can this feature be enabled / disabled in a live cluster?
```

Comment:
Further down in the doc, please:
1. Answer the question `###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?`.
2. `###### What are other known failure modes?` should probably list the feature enablement ordering issue. 
3. `###### What steps should be taken if SLOs are not being met to determine the problem?` needs an answer
4. `Implementation history` needs update

Reply:
for (1), what kind of test should I actually perform at each stage of upgrade->downgrade->upgrade again? given this touches a subset of tokens that can be used to authenticate to an apiserver, is it sufficient to just test that these credentials are valid/invalid depending on the upgrade/downgrade (or enablement) status? Is simply enabling the gates and then disabling them again sufficient, or would you expect a full apiserver upgrade to be performed?

for (2), I've not included this as I've added an additional check into the apiserver to prevent startup if the wrong combination of gates is enabled (see https://github.com/kubernetes/enhancements/pull/4470#discussion_r1479866040)

(3) and (4) have been completed

thanks!

Reply:
> for (1), what kind of test should I actually perform at each stage of upgrade->downgrade->upgrade again? given this touches a subset of tokens that can be used to authenticate to an apiserver, is it sufficient to just test that these credentials are valid/invalid depending on the upgrade/downgrade (or enablement) status? Is simply enabling the gates and then disabling them again sufficient, or would you expect a full apiserver upgrade to be performed?

yup, that sounds sufficient. 

> for (2), I've not included this as I've added an additional check into the apiserver to prevent startup if the wrong combination of gates is enabled (see [#4470 (comment)](https://github.com/kubernetes/enhancements/pull/4470#discussion_r1479866040))

thank you!

> (3) and (4) have been completed

thank you!